go.sum

```diff
@@ -224,8 +224,8 @@ github.com/mattermost/ldap v0.0.0-20201202150706-ee0e6284187d h1:/RJ/UV7M5c7L2TQ
 github.com/mattermost/ldap v0.0.0-20201202150706-ee0e6284187d/go.mod h1:HLbgMEI5K131jpxGazJ97AxfPDt31osq36YS1oxFQPQ=
 github.com/mattermost/logr/v2 v2.0.15 h1:+WNbGcsc3dBao65eXlceB6dTILNJRIrvubnsTl3zBew=
 github.com/mattermost/logr/v2 v2.0.15/go.mod h1:mpPp935r5dIkFDo2y9Q87cQWhFR/4xXpNh0k/y8Hmwg=
-github.com/mattermost/mattermost-plugin-api v0.1.1 h1:bNnPbWCLWZpT/k2kjUxNnzCfUggU8WKs2ddz7hNjg1U=
-github.com/mattermost/mattermost-plugin-api v0.1.1/go.mod h1:9yZhtg0bBj3kqSTjXnjYBMZoTsWbe3ajdFMdl9/Jz34=
+github.com/mattermost/mattermost-plugin-api v0.1.3 h1:Z5KBAV/Lpsf1NleSRSeun3jPQOIvN3/G+Xw5DqB2rac=
```

Comment:
I'm a bit surprised you didn't need to pin the exact commit since I didn't tag a new release. Example at [github plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/go.mod#L9
).

Reply:
Sorry, I pinned the comment in the go.sum... my comment was related to [go.mod](https://github.com/mattermost/mattermost-plugin-gitlab/pull/365/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R8)

Reply:
it doesn't seem to be correct (and has my name on it), I don't remember creating a tag, and the linked commit is an orphan (part of a pr commit).